internal/execute/tsc.go

```diff
@@ -41,25 +41,20 @@ func applyBulkEdits(text string, edits []core.TextChange) string {
 	return b.String()
 }
 
-func CommandLine(sys System, cb cbType, commandLineArgs []string) ExitStatus {
+func CommandLine(sys System, commandLineArgs []string, testing bool) (ExitStatus, *incremental.Program, *Watcher) {
```

Comment:
Is it time to start returning a struct here? I assume that this is all just here because testing needs more info out of the CommandLine function?